webapp/src/components/CalendarPage.tsx

```diff
@@ -24,23 +32,25 @@ const CalendarPage = () => {
   if (!programs) {
     return null;
   }
+  const programEventsActivities: RBCEvent[] = programs[0].activities.map(
+    activity => {
+      return {
+        title: activity.title,
+        start: new Date(activity.start_time),
+        end: new Date(activity.end_time),
+        description: activity.description_text,
```

Comment:
Do we need this line in Calendar? I think we didn't use it.

Reply:
That is the information that is feeding the <Calendar>

Reply:
Thank you, Cami, the explanation!